docs/source/cuml_intro.rst

```diff
@@ -1,36 +1,57 @@
 Introduction
 ============
 
-cuML accelerates machine learning on GPUs. The library follows a
-couple of key principles, and understanding these will help you take
-full advantage cuML.
-
+cuML's design is built around three core principles that work together to make
+GPU-accelerated machine learning both powerful and accessible. These principles
+guide every aspect of the library's development and determine how you interact
+with cuML in practice. Understanding these principles will help you take full
+advantage of cuML's capabilities and write more effective code.
 
 1. Where possible, match the scikit-learn API
 ---------------------------------------------
 
 cuML estimators look and feel just like `scikit-learn estimators
-<https://scikit-learn.org/stable/developers/develop.html>`_. You
-initialize them with key parameters, fit them with a ``fit`` method,
-then call ``predict`` or ``transform`` for inference.
+<https://scikit-learn.org/stable/developers/develop.html>`_. This means:
 
+* **Familiar workflow**: Initialize with parameters, fit with data, predict/transform for inference
+* **Drop-in replacement**: Most sklearn code works with minimal changes
+* **Consistent naming**: Same method names and parameter conventions
+* **Compatible interfaces**: Works with sklearn's cross-validation, pipelines, and preprocessing
 
 .. code-block:: python
 
-   import cuml.LinearRegression
+   from cuml import LinearRegression
+   from cuml.datasets import make_regression
+   import numpy as np
+
+   # Generate sample data
+   X, y = make_regression(n_samples=1000, n_features=4, noise=0.1, random_state=42)
+
+   # Split data (same as sklearn)
+   from sklearn.model_selection import train_test_split
+   X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
 
-   model = cuml.LinearRegression()
-   model.fit(X_train, y)
-   y_prediction = model.predict(X_test)
+   # Fit model (same API as sklearn)
+   model = LinearRegression()
+   model.fit(X_train, y_train)
+
+   # Make predictions
+   y_pred = model.predict(X_test)
+   print(f"R² score: {model.score(X_test, y_test):.3f}")
 
 You can find many more complete examples in the `Introductory Notebook
 <estimator_intro.ipynb>`_ and in the cuML API documentation.
 
+.. note::
+
+   Use `cuml.accel <cuml-accel/index.rst>`_ if you need 100% API
+   parity with scikit-learn.
+
 2. Accept flexible input types, return predictable output types
 ---------------------------------------------------------------
 
 cuML estimators can accept NumPy arrays, cuDF dataframes, cuPy arrays,
-2d PyTorch tensors, and really any kind of standards-based Python
+2D PyTorch tensors, and really any kind of standards-based Python
 array input you can throw at them. This relies on the ``__array__``
```

Comment:
```suggestion
and 2D PyTorch tensors. This relies on the ``__array__``
```

I'd make it specific instead of being vague and using informal language. In particular because the most obvious thing to try, lists, doesn't work.